.gitignore

```diff
@@ -165,4 +165,6 @@ build/
 .envs/
 
 # licheck artifacts
-.requrements.txt
+.requirements.txt
+
+temp/
```

Comment:
what's the reason for adding an arbitrary directory as a gitignore? as far as I see no such directory is being used by the code in this repo ...